src/Array2XML.php

```diff
@@ -73,13 +83,18 @@ public static function createXML($node_name, $arr = [], $docType = [])
      * @param string $encoding
      * @param bool   $standalone
      * @param bool   $format_output
+     * @param string $cdata
+     * @param string $value
+     * @param string $attributes
      */
-    public static function init($version = '1.0', $encoding = 'utf-8', $standalone = false, $format_output = true)
+    public static function init($version = '1.0', $encoding = 'utf-8', $standalone = false, $format_output =true, $cdata = '@cdata', $value = '@value', $attributes = '@attributes')
```

Comment:
Maybe make the string literals constants in a new consts only class so that they can be shared without duplication? (Picking a bit picky on this ... just thinking if further reduces the duplication).